bundler/lib/bundler/source/git.rb

```diff
@@ -11,6 +11,7 @@ class Git < Path
 
       def initialize(options)
         @options = options
+        @checksum_store = Checksum::Store.new
```

Comment:
I don't know if this is right, but the cases where store was nil on source.checksum_store.merge! were when the source was Git.

Reply:
@segiddins Do you understand this part enough to know if this is fine? There a place in the code where it tries to merge lock checksums into this git source and there's no checksum store. Original code used checksum_store&.use to avoid the problem, but it seems weird to lose checksums that we have. Thoughts?

Reply:
yeah this is fine. I dont think we'd lose checksums in practice because we would have checksums ever for git source gems, but probably better to be consistent here 👍🏻